paint/transaction-payment/src/lib.rs

```diff
@@ -56,7 +56,7 @@ type NegativeImbalanceOf<T> =
 
 pub trait Trait: system::Trait {
 	/// The currency type in which fees will be paid.
-	type Currency: Currency<Self::AccountId>;
+	type Currency: Currency<Self::AccountId> + Send + Sync;
```

Comment:
Can you elaborate why is `Send + Sync` required? I wouldn't expect to see that in any part of runtime.

Reply:
Good point, I didn't really care much since I have seen these two in all SignedExtension type. For instance: https://github.com/paritytech/substrate/blob/0f87c92d2fb4bd7d9c235b1fb0caffc2ce58f803/paint/system/src/lib.rs#L853

So I assumed it is okay. But I will look it up. 

although, we use `Send + Sync` in `Member`  which is used around the runtime, so just by observation, I would not assume that we ban them in runtime. Although it does makes sense that a single threaded runtime should not really need them.

Reply:
Needed due to `AccountId` being `Member`.